### ISSUE TITLE
[FEATURE REQUEST] Loosen Release version poetry requirements

### ISSUE THREAD

**Issue by encodexdev** [getzep/zep-python#136]
Could you lower the `httpx` requirement to "^0.24.0" for the release versions of the build?

My project requires `supabase` python which doesn't have a package version that supports a version this high.

I also had to downgrade the `pytest-httpx` to "^0.24.0"

Below is the updated poetry file:

```
[tool.poetry]
name = "zep-python"
version = "2.0.0-rc.3"
description = "Zep: Fast, scalable building blocks for LLM apps. This is the Python client for the Zep service."
authors = ["Daniel Chalef <daniel.chalef@private.org>"]
readme = "README.md"

[tool.poetry.dependencies]
python = ">=3.9.0,<4"
httpx = "^0.24.0"
pydantic = ">=2.0.0"

[tool.poetry.group.test.dependencies]
pytest = "^7.3.1"
pytest-httpx = "^0.24.0"
pytest-asyncio = "^0.21.0"
ruff = "^0.0.291"
mypy = "^1.2.0"
pytest-mock = "^3.10.0"

[tool.poetry.group.dev.dependencies]
packaging = "^23.1"
black = "^24.2.0"
ruff = ">=0.0.291"
mypy = "^1.2.0"
pre-commit = "^3.3.1"
pytest-httpx = "^0.24.0"
faker = "^19.2.0"
python-dotenv = "^1.0.0"
pdoc3 = "^0.10.0"
langchain = "^0.1.3"
openai = "^1.9.0"
langchain-openai = "^0.0.3"
langchain-cli = { extras = ["serve"], version = "^0.0.21" }
bs4 = "^0.0.2"

[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"

```

**Comment by danielchalef** [comment]
@ellipsis please create a pr to downgrade httpx and httpx-pytest to version "^0.24.0".

@encodexdev thanks for reporting this. Are you using Zep Open Source or Zep Cloud?

**Comment by encodexdev** [comment]
https://github.com/getzep/zep-python/pull/137

I created a PR to help. I am using zep cloud.

**Comment by danielchalef** [comment]
Thank you! I'll be merging #138, which resolves a merge conflict in #137. Appreciate your assistance here!
